lib/rules/indent.js

```diff
@@ -0,0 +1,832 @@
+/**
```

Comment:
Running this new indent rule against this repo reveals a lot of indentation errors, both in this new file and in a couple of older files. Please fix these. 

You can run this rule against the repo by using `npm link` and then `npm link eslint-plugin-silvermine` to link this repo to itself (repeating the process for a version of `eslint-config-silvermine` that contains configuration that enables this rule) and then running `grunt standards`.